### ISSUE TITLE
crashes randomly

### ISSUE THREAD

**Issue by oltic** [mpurses/Sonder#5]
basically ill be running this skin then it will just stop, im on wind 7 and it tells me that it just ran into a problem but dosnt say what it is

**Comment by mpurses** [comment]
Can you give me a list of all the skins you have currently loaded? And on the settings tab of rainmeter can you select 'Debug mode' and 'Log to file' so when it crashes again you can check the log file to see what the last thing rainmeter tried to do was.

**Comment by oltic** [comment]
the only skin I use is this one, also I'm not sure what I'm looking for in the log file. one thing that pops up is it can't find "A GPU Catagory". and when it crashes, the program just gets closed by windows so I cant check the log, basically deleting everything before I start it up again.

**Comment by mpurses** [comment]
By skin I mean a list of all the individual skins loaded (see image). And for the log file, first turn on logging to a file(see second image), so when it crashes you can go to the file and see what happened right before it crashed. You don't have to understand the log file, you can just post it here and I'll try to make sense of it.  It will be located here: C:\Users\<user>\AppData\Roaming\Rainmeter\Rainmeter.log

![image](https://user-images.githubusercontent.com/642218/96273619-27f76300-0f95-11eb-9938-df76306892b6.png)

![image](https://user-images.githubusercontent.com/642218/96274009-a227e780-0f95-11eb-8656-f5e7d359b961.png)

**Comment by oltic** [comment]
hello again, the skins I have loaded include WeatherLive.ini, Brightness.ini, Volume.ini, System.ini, 1-Date_hori1.ini, Time_hori1.ini.
here is the log 
![ahhhhhhhh](https://user-images.githubusercontent.com/72896438/96331600-cf1edd80-101b-11eb-8170-e5cec5ded98d.PNG)

**Comment by mpurses** [comment]
Is this at the point where it crashes? 
I'm not sure if your system has a dedicated GPU, if not I would uncheck (hide) the GPU option for the System skin. Because it doesn't look like it is finding one.

**Comment by oltic** [comment]
yes that is the point when it stops, I also unchecked that box and the program still crashed here's a photo of the message it says 
![ahhh2](https://user-images.githubusercontent.com/72896438/96354707-dc2ce280-1096-11eb-98ff-263752f0b789.PNG)

It's odd, even with it turned off it still happens. I'm starting to think this could just be a problem with the program its self

**Comment by mpurses** [comment]
I would start unloading one skin at a time (probably starting with the system one) to see if it stops crashing to narrow down which one it could be.

**Comment by mpurses** [comment]
I would also download the latest Beta just in case that fixes something...
https://www.rainmeter.net/

**Comment by oltic** [comment]
I unloaded skins slowly over time and I've found that the problem came from brightness.ini

**Comment by mpurses** [comment]
Thank you for finding that out. I had another user run into the brightness skin causing crashing as well. (That's why I don't have it installed by default in the layouts.)  Could you tell me some info about the graphics on your system?

Device Manager > Display adapters

![Screenshot 2020-11-16 095156](https://user-images.githubusercontent.com/642218/99275660-63ce5400-27f1-11eb-8758-0d3520f0b4d8.png)

I didn't write the brightness plugin but I might be able to get the author to fix it if I have info on the graphics hardware that crashes it.

**Comment by mpurses** [comment]
And if you wanted to see if another version with a different plugin works try this one:

https://forum.rainmeter.net/viewtopic.php?f=18&t=18506

If that one doesn't crash your system I can try incorporating that plugin into the brightness skin as an option.
